src/node_sea.cc

```diff
@@ -161,6 +205,60 @@ void IsExperimentalSeaWarningNeeded(const FunctionCallbackInfo<Value>& args) {
       sea_resource.flags & SeaFlags::kDisableExperimentalSeaWarning));
 }
 
+void GetCodeCache(const FunctionCallbackInfo<Value>& args) {
+  if (!IsSingleExecutable()) {
+    return;
+  }
+
+  Environment* env = Environment::GetCurrent(args);
+  Isolate* isolate = env->isolate();
+  HandleScope scope(isolate);
+
+  SeaResource sea_resource = FindSingleExecutableResource();
+
+  if (!static_cast<bool>(sea_resource.flags & SeaFlags::kUseCodeCache)) {
+    return;
+  }
+
+  std::shared_ptr<BackingStore> backing_store = ArrayBuffer::NewBackingStore(
+      const_cast<void*>(
+          static_cast<const void*>(sea_resource.code_cache->data())),
+      sea_resource.code_cache->length(),
+      [](void* /* data */, size_t /* length */, void* /* deleter_data */) {
+        // The code cache data string is not freed here because it is a static
+        // string which is not allocated by the BackingStore allocator.
```

Comment:
```suggestion
        // The code cache data blob is not freed here because it is a static
        // blob which is not allocated by the BackingStore allocator.
```